translations/copyq_it.ts

```diff
@@ -271,176 +271,176 @@ Potete impostare il comando nelle preferenze.</translation>
     <message>
         <location filename="../src/ui/addcommanddialog.ui" line="14"/>
         <source>CopyQ Add Commands</source>
-        <translation type="unfinished"></translation>
+        <translation>Comandi Aggiungi CopyQ</translation>
     </message>
     <message>
         <location filename="../src/gui/addcommanddialog.cpp" line="81"/>
         <source>Ctrl+Shift+1</source>
         <comment>Global shortcut for some predefined commands</comment>
-        <translation type="unfinished"></translation>
+        <translation>CTRL+Miausc+1</translation>
```

Comment:
Seems that this shortcut is not recognized by the Qt GUI framework. When I try to map the shortcut in the app it still shows as "Ctrl+Shift+1" so it should probably be the same in the translation.

The other translated shortcut ("CTRL+SPAZIO") works fine.

BTW, you can test the translation on Windows by clicking on details link next to "continuous-integration/appveyor/pr — AppVeyor build succeeded" (after the app is successfully built)

> ![image](https://user-images.githubusercontent.com/38054/27803578-711d984c-602a-11e7-9a13-6378d37095a6.png)

clicking on the first job name

> ![image](https://user-images.githubusercontent.com/38054/27803625-b54636c8-602a-11e7-962b-3102d103c8e4.png)

selecting ARTIFACTS and downloading portable app in zip file.

> ![image](https://user-images.githubusercontent.com/38054/27803669-f86dc72c-602a-11e7-9482-1796287c17e1.png)